sdk/nodejs/tests/runtime/tsClosureCases.ts

```diff
@@ -5990,22 +5990,22 @@ return function () { typescript.parseCommandLine([""]); };
         });
     }
 
-    {
-        cases.push({
-            title: "Fail to capture non-deployment module due to native code",
-            func: function () { console.log(pulumi); },
-            error: `Error serializing function 'func': tsClosureCases.js(0,0)
-
-function 'func':(...)
-  module './bin/index.js' which indirectly referenced
-    function 'debug':(...)
-(...)
-Function code:
-  function (...)() { [native code] }
-
-Module './bin/index.js' is a 'deployment only' module. In general these cannot be captured inside a 'run time' function.`
-        });
-    }
+//     {
+//         cases.push({
+//             title: "Fail to capture non-deployment module due to native code",
+//             func: function () { console.log(pulumi); },
+//             error: `Error serializing function 'func': tsClosureCases.js(0,0)
+
+// function 'func':(...)
+//   module './bin/index.js' which indirectly referenced
+//     function 'debug':(...)
+// (...)
+// Function code:
+//   function (...)() { [native code] }
+
+// Module './bin/index.js' is a 'deployment only' module. In general these cannot be captured inside a 'run time' function.`
+//         });
+//     }
```

Comment:
technically we don't have any native code anymore!  i need a way to test this in a non-hacky fashion.